.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-sudo: false
+sudo: true
```

Comment:
We need this to stay how it is, see [Migrating from legacy to container-based infrastructure](https://docs.travis-ci.com/user/migrating-from-legacy/#tl%3Bdr)